<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcome</class>
 <widget class="QMainWindow" name="welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>welcome</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1, 116, 190);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="backgroundFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1151</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>375</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>750</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Persian">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 53, 106);
color: rgb(255, 196, 54);</string>
    </property>
    <property name="text">
     <string>فارسی</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_English">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 53, 106);
color: rgb(255, 196, 54);</string>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P_Register">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
color: rgb(255, 196, 54);
background-color: rgb(12, 53, 106);</string>
    </property>
    <property name="text">
     <string>همین حالا ثبت نام کنید</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P_LonIn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
color: rgb(255, 196, 54);
background-color: rgb(12, 53, 106);</string>
    </property>
    <property name="text">
     <string>قبلا ثبت نام کردید وارد شوید</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_E_Register">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>520</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
color: rgb(255, 196, 54);
background-color: rgb(12, 53, 106);</string>
    </property>
    <property name="text">
     <string>register now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_E_LonIn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>520</y>
      <width>391</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
color: rgb(255, 196, 54);
background-color: rgb(12, 53, 106);</string>
    </property>
    <property name="text">
     <string>You have already registered, enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P_Goals">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>40</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
color: rgb(255, 196, 54);
background-color: rgb(12, 53, 106);</string>
    </property>
    <property name="text">
     <string>اهداف مجموعه</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_E_Goals">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
color: rgb(255, 196, 54);
background-color: rgb(12, 53, 106);
</string>
    </property>
    <property name="text">
     <string>Objectives of the collection</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
